KMymoney tries to find qgpgme before kdepimlibs. This is wrong because
FindQGpgme is installed by kdepimlibs, thus can be invisible until kdepimlibs
found.
diff --git a/CMakeLists.txt b/CMakeLists.txt
index f6d7305..88bac67 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,8 +59,8 @@ else (WIN32)
   find_package(Boost "1.33.1" COMPONENTS graph)
 endif (WIN32)
 # needed by libkgpgfile
-find_package(QGpgme REQUIRED)
 find_package(KdepimLibs REQUIRED)
+find_package(QGpgme REQUIRED)
 find_package(SharedMimeInfo REQUIRED)
 
 add_definitions( ${QT_DEFINITIONS} ${KDE4_DEFINITIONS} ${KDEPIM_DEFINITIONS})
